options/locale/locale_en-US.ini

```diff
@@ -118,6 +118,20 @@ footer = Footer
 footer.software = About Software
 footer.links = Links
 
+[editor]
+header.tooltip = Add heading
+bold.tooltip = Add bold text
+italic.tooltip = Add italic text
+quote.tooltip = Quote text
+code.tooltip = Add code/highlighted text
```

Comment:
```suggestion
code.tooltip = Add code
```

Reply:
The problem I can see with that:
I think `code` is surprisingly unspecific.
Especially since it is basically just text highlighting.
When I read it myself, I have no idea what the result of clicking that button will be, thus making the tooltip useless.
What about the opposite?
```suggestion
code.tooltip = Add highlighted text
```

Reply:
Don't you agree that essentially all stuff in backticks is "code"?